module/VuFind/src/VuFind/ILS/Driver/KohaILSDI.php

```diff
@@ -980,6 +981,7 @@ public function getHolding($id, array $patron = null)
                     ? 'Unknown' : $rowItem['BARCODE'],
                 'number'       => (null == $rowItem['COPYNO'])
                     ? '' : $rowItem['COPYNO'],
+                'enumchron'    => $rowItem['ENUMCHRON'] ?? '',
```

Comment:
I would suggest defaulting to null here rather than '' -- the empty string will evaluate to true when passed to isset, causing an empty value to be displayed in the template.

Reply:
I've set this to default to null now, so the Voyager and KohaILSDI drivers should now be consistent.